package.json

```diff
@@ -76,6 +76,7 @@
     "html-webpack-plugin": "^3.2.0",
     "husky": "^4.2.3",
     "node-sass": "^4.14.1",
+    "postcss-copy-assets": "^0.3.1",
```

Comment:
Is this this the only package out there? This hasn't been touched in over 3 years. Dependencies we add need to be actively maintained, otherwise we are accountable and get bombarded with security vulnerabilities from InfoSec.

Reply:
I didn't notice that the package is not maintained anymore. I replace the package with the `postcss-url` which does approximately the same thing.

Reply:
Thanks for finding an alternative